.golangci.yml

```diff
@@ -56,16 +56,13 @@ linters-settings:
       - commentFormatting
 
   gofumpt:
-    lang-version: *goVersion
```

Comment:
is this effectively changing the checks? would it be better to update `goVersion` up in this file?

Reply:
Yeah, that's something I forgot to update when upgrading to Go 1.23. Just changed it, let's see if all the lints still pass.

Reply:
@exekias All looks good now

Reply:
do we need to remove this line now then?

Reply:
I removed those lines because they were being flagged as deprecated by `golangci-lint`